libtsuba/src/RDGSlice.cpp

```diff
@@ -20,7 +20,7 @@ tsuba::RDGSlice::DoMake(
 
   auto node_result = AddPropertySlice(
       metadata_dir, core_->part_header().node_prop_info_list(),
-      slice.node_range,
+      slice.node_range, nullptr, /* XXX */
```

Comment:
Are these comments meaningful? Temporary?

Reply:
oh yeah, that's a note to self to make sliced reads async as well.